static/sass/styles.scss

```diff
@@ -1659,3 +1664,10 @@ ol.ordered-counter {
     margin-right: 0.25rem;
   }
 }
+
+legend {
```

Comment:
Could this extend the class we have [on vanilla](https://github.com/canonical/vanilla-framework/blob/a0d8f348e9bcd8f9f523eb356f95ca09b149f41d/scss/_base_forms.scss#L101) or better yet, could we make a push to upstream this? cc: @bartaz

Reply:
Just pushed a PR to upstream this to vanilla!